### PR TITLE
Include led test from base

### PR DESCRIPTION
**Pull request by rickwu666666**
Since we have migrated LED cases back to the checkbox base, we need to remove the existing job and nest the plan from the checkbox base.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,9 +119,9 @@ GPIO_LEDS={name1}:{port1} {name2}:{port2} ...
 e.g. GPIO_LEDS=dl1:488 dl2:489 dl44:507
 
 id: ce-oem-sysfs-leds
-SYS_LEDS={path1}:{position1} {path2}:{position1} ...
+SYS_LEDS={position1}:{path1} {position1}:{path2} ...
```
Comment (stanley31huang) [inline README.md:122]: should be `SYS_LEDS={position1}:{path1} {position2}:{path2} ...`?

**Review by stanley31huang — CHANGES_REQUESTED** [review]
there's a minor issue in this PR, please see my inline comment

**Comment by rickwu666666** [comment]
> there's a minor issue in this PR, please see my inline comment

@stanley31huang I've fixed it. Thanks for the review.

**Review by stanley31huang — APPROVED** [review]
LGTM

**Review by patliuu — APPROVED** [review]
LGTM

**Pull request merged**